pennylane/ops/op_math/pow.py

```diff
@@ -262,6 +262,11 @@ def has_decomposition(self):
             self.base.pow(self.z)
         except PowUndefinedError:
             return False
+        except Exception as e:  # pylint: disable=broad-except
+            # some Pow methods cant handle a batched z
```

Comment:
Just to send possible readers onto the right track :)
```suggestion
            # some pow methods cant handle a batched z
```